src/actions/index.js

```diff
@@ -199,3 +203,19 @@ export function changeEventResponse (id, response, user) {
 export function toggleMainMenu () {
   return {type: TOGGLE_MAIN_MENU}
 }
+
+export function updateUserSettings (params, prevProps) {
+  return {
+    type: UPDATE_USER_SETTINGS,
+    payload: {api: true, params, path: `/noo/user/${params.id}`, method: 'POST'},
+    meta: {params, prevProps}
+  }
+}
+
+export function leaveCommunity (communityId, prevProps) {
+  return {
+    type: LEAVE_COMMUNITY,
+    payload: {api: true, path: `/noo/membership/${communityId}`, method: 'DELETE'},
+    meta: {communityId: communityId, prevProps}
```

Comment:
you can just do `{communityId, ...}` instead of `{communityId: communityId, ...}`